### DISCUSSION TITLE
The 90mph office - Alan Pope's blog

### DISCUSSION THREAD

**Discussion by giscus** [popey/popey.com-blog#35] [General]
# The 90mph office - Alan Pope's blog

Lightbulb moment On Thursday, October 13th 2016, while on a train, slowly recovering from a pub-based night out, I had an idea!
The idea is to take a train somewhere (unspecified) for four hours, have lunch, then take another train home. The train would be my 90mph office with scenery for those four hours in each direction, and I’d get to enjoy lunch somewhere new. I’d do a full days work with the view out the window constantly changing.

https://popey.com/blog/2023/09/the-90mph-office/

**Comment by CodingHerbivore** [comment]
This sounds like something CGP Grey would do.

I love the idea. I would think it would need to be a route you have done before though, lest you get constantly distracted by the scenery. Bathroom breaks may be a nuisance too, as you'd want to pack up your laptop and any other things around you.

**Reply by popey** [reply]
I dunno about knowing the route. I quite like the idea of staring out the window and new countryside passing by. Everyone needs a *little* distraction through the day, to rest the eyes and maybe think a bit. Bio-breaks is certainly a concern. I'm hoping the laptop is the only thing I'll have out, with bluetooth headphones and phone in pocket maybe. We shall see!
